<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>左上点坐标：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="x0"/>
      </item>
      <item>
       <widget class="QLineEdit" name="y0"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>右下点坐标：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="x1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="y1"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ImgRootPath：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="RootPath"/>
      </item>
      <item>
       <widget class="QPushButton" name="before_button">
        <property name="text">
         <string>上一张图片</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="after_button">
        <property name="text">
         <string>下一张图片</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>SavePath：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="SavePath"/>
      </item>
      <item>
       <widget class="QPushButton" name="Save_button">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
